tests/testthat/test-shinytest2-tm_a_pca.R

```diff
@@ -0,0 +1,171 @@
+testthat::test_that("e2e - tm_a_pca: data extract changes output", {
+  skip_if_too_deep(5)
+
+  require(shinytest2)
+
+  data <- within(teal_data(), {
+    require(nestcolor)
+    require(ggplot2)
+
+    USArrests <- USArrests # nolint: object_name.
+  })
+  datanames(data) <- "USArrests"
+
+  app <- TealAppDriver$new(
```

Comment:
```suggestion
  app <- teal::TealAppDriver$new(
```

Reply:
ah, just seeing this being imported in `tests/testthat/helper-TealAppDriver.R`. ignore please